apps/aehttp/test/aehttp_spec_SUITE.erl

```diff
@@ -73,3 +75,31 @@ get_api(Config) ->
     {ok,{{"HTTP/1.1",405,"Method Not Allowed"},_,_}} =
          httpc:request(post, {URL, [], Type, Body}, [], []),
     ok.
+
+validate_api(Config) ->
+    Spec = json_from_yaml(Config),
+
+    Url = "http://validator.swagger.io/validator/",
+    {ok, {_, _Headers, Body}} = httpc:request(post, {Url, [],  "application/json", Spec}, [], []),
```

Comment:
Maybe handling some more responses (and indicate that there is a connection problem/remote problem)?